### PR TITLE
Fix tile map atlas loading when the image contains unused pixels at the right or bottom.

### PR DESCRIPTION
**Pull request by D8H**
It fixes this issue
* https://forum.gdevelop.io/t/tilemap-not-rendering/40955/7

### CODE REVIEW

File: SharedLibs/TileMapHelper/src/render/TileMapPixiHelper.ts [hunk 1]
```diff
@@ -155,11 +169,14 @@ export class PixiTileMapHelper {
       } else if (layer instanceof EditableTileMapLayer) {
         const tileLayer = layer as EditableTileMapLayer;
 
+        const tileWidth = tileLayer.tileMap.getTileWidth();
+        const tileHeight = tileLayer.tileMap.getTileHeight();
+
         for (let y = 0; y < tileLayer.tileMap.getDimensionY(); y++) {
+          let first = true;
```
Comment (ClementPasteau) [inline SharedLibs/TileMapHelper/src/render/TileMapPixiHelper.ts:176]: this seems unused?

File: SharedLibs/TileMapHelper/src/render/TileMapPixiHelper.ts [hunk 2]
```diff
@@ -155,11 +169,14 @@ export class PixiTileMapHelper {
       } else if (layer instanceof EditableTileMapLayer) {
         const tileLayer = layer as EditableTileMapLayer;
 
+        const tileWidth = tileLayer.tileMap.getTileWidth();
+        const tileHeight = tileLayer.tileMap.getTileHeight();
+
         for (let y = 0; y < tileLayer.tileMap.getDimensionY(); y++) {
```
Comment (ClementPasteau) [inline SharedLibs/TileMapHelper/src/render/TileMapPixiHelper.ts:175]: should the dimensionY and dimensionX be calculated outside the loops too? Or can they change?

**Review by ClementPasteau — COMMENTED** [review]
lgtm!

File: SharedLibs/TileMapHelper/src/render/TileMapPixiHelper.ts (hunk 1 above)
Comment (D8H) [reply SharedLibs/TileMapHelper/src/render/TileMapPixiHelper.ts:176]: oh, yes, it was for debugging.
good catch

File: SharedLibs/TileMapHelper/src/render/TileMapPixiHelper.ts (hunk 2 above)
Comment (D8H) [reply SharedLibs/TileMapHelper/src/render/TileMapPixiHelper.ts:175]: I could move them.

**Review by ClementPasteau — APPROVED** [review]
👍

**Pull request merged**